website/hugo.yaml

```diff
@@ -76,13 +76,13 @@ params:
         url: "https://slack.k8s.io/"
         icon: fab fa-slack
         desc: "Chat with us on Slack in the #aws-provider channel"
-  latest_release_version: v0.34
+  latest_release_version: v0.35.0
```

Comment:
This latest release version still doesn't look correct. Do we need to make manual changes here to make this look correct?

Reply:
Looking at where this is actually referenced this change actually looks correct, it drives the KARPENTER_VERSION env var in the preview docs.